profiler/src/main/java/com/splunk/opentelemetry/profiler/Configuration.java

```diff
@@ -51,6 +55,7 @@
     config.put(CONFIG_KEY_RECORDING_DURATION, DEFAULT_RECORDING_DURATION);
     config.put(CONFIG_KEY_KEEP_FILES, "false");
     config.put(CONFIG_KEY_MEMORY_ENABLED, String.valueOf(DEFAULT_MEMORY_ENABLED));
+    config.put(CONFIG_KEY_CALL_STACK_INTERVAL, DEFAULT_CALL_STACK_INTERVAL + " ms");
```

Comment:
Seems like it relies on `Duration.toString()` method. Probably explicit conversion to ms is more reliable.

Reply:
Wont' `Duration.toString()` produce sth like `PT10S` by default?

Reply:
Yeah good catch. I changed the type (it was just a long before) and this got missed. Will fix it.